package/gluon-core/luasrc/lib/gluon/upgrade/140-firewall-rules

```diff
@@ -81,8 +81,17 @@ end
 
 
 -- We can't put mesh_wan into this zone, as mesh_wan is the same
--- interface as wan, which has its own zone
-uci:set('firewall', 'wired_mesh', 'network', {'mesh_lan'})
+-- interface as wan, which has its own zone but allow any other defined
+-- gluon_wired interface addicted to bat0 to communicate
+mil = {}
```

Comment:
- Missing `local` (if you have `lualint` installed, you can `make lint-lua` to find such issues)
- Let's name this `wired_mesh_ifaces`